network/src/peer_registry.rs

```diff
@@ -186,14 +175,11 @@ impl PeerRegistry {
         let mut rng = thread_rng();
         evict_group.shuffle(&mut rng);
 
-        // randomly evict a lowest scored peer
-        evict_group
-            .iter()
-            .min_by_key(|peer| peer_store.peer_score(&peer.peer_id).unwrap_or_default())
-            .map(|peer| {
-                debug!(target: "network", "evict inbound peer {:?}", peer.peer_id);
-                peer.session_id
-            })
+        // randomly evict a peer
+        evict_group.get(0).map(|peer| {
```

Comment:
use `choose`, no need to shuffle.

```suggestion
        evict_group.choose(&mut rng).map(|peer| {
```